src/Morpho.sol

```diff
@@ -153,14 +153,10 @@ contract Morpho is IMorphoStaticTyping {
         require(isLltvEnabled[marketParams.lltv], ErrorsLib.LLTV_NOT_ENABLED);
         require(market[id].lastUpdate == 0, ErrorsLib.MARKET_ALREADY_CREATED);
 
-        // Safe "unchecked" cast.
-        market[id].lastUpdate = uint128(block.timestamp);
+        _accrueInterest(marketParams, id);
```

Comment:
Then `AccrueInterest` is logged before `CreateMarket` which is very weird.  I I think we should do the opposite

Reply:
good point, I'm not sure that it's a problem though. It's like that in all functions (I understand that createMarket is different, but i'm not convinced that we should change that)

Reply:
We already had the discussion and decided that it was important to have `AccrueInterest` emitted after `CreateMarket`

Reply:
yes I remember, but I don't think that it is a real problem as it is like that in all functions. If you want I can change it though.

Reply:
> as it is like that in all functions

wdym?

I think we should change it yes since it makes the listening of market creation harder

Reply:
>wdym?

accrue interest is emitted before the event of the function itself

>I think we should change it yes since it makes the listening of market creation harder

why ?

Reply:
Because then `AccrueInterest(id, borrowRate, interest, feeShares)` is logged with an unknown `id` and you need to check the second event `CreateMarket(id, marketParams)` to link this `id` to newly created market which is very counter intuitive.

Reply:
ok, note that you can also rely on accrue interest to check for market creations too

Reply:
sure I had this in mind but I don't know if this is really a problem (as I never actually use events myself). @julien-devatom thoughts?